big_o_exercise/big_o_exercise.js

```diff
@@ -0,0 +1,25 @@
+/* Part 1
+1. O(n)
+2. O(n)
+3. O(1)
+4. O(n^3)
+5. O(n)
+6. O(log n)
```

Comment:
Your answers for 6 - 8 aren't quite right. It may help to plot some of these terms and see how they grow.